tsconfig.json

```diff
@@ -7,6 +7,7 @@
 		"module": "commonjs",
 		"moduleResolution": "node",
 		"esModuleInterop": true,
+    "isolatedModules": true,
```

Comment:
Please use TAB here instead of spaces. Indentation is broken otherwise.

Reply:
Maybe a Prettier action brings everybody easily on the same page ;) https://github.com/marketplace/actions/prettier-action

Reply:
I know prettier, but prefer to not use it in mediasoup-client yet, perhaps in the future.